python/ray/tests/test_advanced_2.py

```diff
@@ -505,6 +508,15 @@ def test_two_custom_resources(ray_start_cluster):
         })
     ray.init(address=cluster.address)
 
+    @ray.remote
+    def foo():
+        time.sleep(0.1)
```

Comment:
Why do we need sleep here? Does it take time for unique_id to be set? If so, can we just wait for condition until unique_id is not nil?